development/tools/jobs/console/react-odata_test.go

```diff
@@ -0,0 +1,72 @@
+package console_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var jobConfigPath = "./../../../../prow/jobs/console/components/react-odata/react-odata.yaml"
+var runIfChangedConfig = "^components/react-odata/"
+var buildScriptPath = "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+var ODataComponentPath = "/home/prow/go/src/github.com/kyma-project/console/components/react-odata"
+
+func TestReactODataJobPresubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig(jobConfigPath)
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Presubmits, 1)
```

Comment:
I don't like the idea of checking number of jobs. If we will add some job in the future then we will need to change every test. Just simply check if expected job exists in:
```
actualPresubmit := tester.FindPresubmitJobByName(jobConfig.Presubmits["kyma-project/console"], expName, "master")
```